File: Backend/Delivery.API/Models/Food.cs
namespace Delivery.API.Models
{
    public class Food
    {
        public int Id { get; set; }

        public int CategoryId { get; set; }
        public Category Category { get; set; }

        public string Name { get; set; }
        public decimal Price { get; set; }
        public decimal? Rating { get; set; }
        public string? Image { get; set; }
        public string? Description { get; set; }
        public int? Calories { get; set; }
        public string? PreparationTime { get; set; }

        // Relaciones
        public ICollection<FoodIngredient> FoodIngredients { get; set; } = new List<FoodIngredient>();
    }

}

File: Frontend/src/app/interfaces/food.ts
import { Ingredient } from './ingredient';

export interface Food {
  id: number;
  categoryId: number;
  name: string;
  price: number;
  rating: number;
  image: string;
  description: string;
  calories: number;
  preparationTime: string;
  ingredients: Ingredient[];
}
